Automattic/wp-calypso
# Framework: Move currentUser Redux state to own root key

This pull request seeks to move current user state from `state.ui.currentUserId` to `state.currentUser.id`, reflecting this change in the folder structure. Despite the number of changes, no functions have changed, nor have any been added or removed.

__Testing instructions:__

Current user is not used by any area of the application, though there are a few pull requests open making  use of it that will need to be updated to use `getCurrentUser` from `state/current-user/selectors` instead of `state/ui/selectors` (#2322, #2472).

Confirm that Mocha tests pass for both current user and UI state by running `make test` from the project root directory or directly from `client/state/current-user` and `client/state/ui`.

/cc @mtias
